qless-java/src/main/java/com/moz/qless/Queue.java

```diff
@@ -23,6 +22,91 @@
 import redis.clients.jedis.Transaction;
 
 public class Queue {
+
+  public static class JobSpec {
```

Comment:
Yeah, it's a bit of a stretch. I had wanted to avoid making all the getters and setters, but I guess the alternative would be making all these members package-protected?